examples/azure-vm-simple-linux-managed-disk/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "hostname" {
+  value = "${var.hostname}"
+}
+
+output "vm_fqdn" {
+  value = "${azurerm_public_ip.pip.fqdn}"
+}
+
+output "sshCommand" {
```

Comment:
to match the terraform naming conventions, it might be best to name this `ssh_command`?